docs/content/what-are-the-differences-between-temporal-and-a-traditional-system.md

```diff
@@ -0,0 +1,65 @@
+---
+id: what-are-the-differences-between-temporal-and-a-traditional-system
+title: What are the differences between Temporal and a traditional system?
+description: Compare a function execution in a traditional system to a Workflow Execution in Temporal.
+tags:
+  - explanation
+---
+
+import DetermineHeader from '../components/DetermineHeader.js'
+
+export const headingText = 'What are the differences between Temporal and a traditional system?'
+
+<DetermineHeader
+hLevel={props.heading}
+hText={headingText}
+/>
+
+The Temporal platform exists to facilitate Workflow Executions.
+Compare a function execution in a traditional system to a Workflow Execution in Temporal.
+
+![Diagram that shows traditional function execution and Temporal Workflow Execution](/img/docs/function-execution-vs-reentrant-process.png)
+
+:::note Legend
+
+λ = traditional function execution
+π = Temporal Workflow Execution
+𝝈 = state
```

Comment:
In the deploy preview, these are not on newlines. They are all on the same line -
Either separate by one more newline or use bullets I think.

Reply:
Oops. Added two spaces at end of first and second lines, which in Markdown translates to a newline without starting a new para.